DataScience/AisAudioLabeler.py

```diff
@@ -512,10 +514,13 @@ def plot_histogram(ais, max_n_ships):
     plt.show()
 
 
-def export_audio_clips(ais, hmd, shp, data_home, hydrophone, clip_home, max_n_ships):
+def export_audio_clips(
+    ais, hmd, shp, data_home, hydrophone, clip_home, max_n_ships, max_distance
+):
     """Export audio clips from AIS intervals during which the
-    specified maximum number of ships are underway. Label the audio
-    clip using the attributes of the ship closest to the hydrophone.
+    specified maximum number of ships at the specified maximum distance are reporting
+    their status as underway. Label the audio clip using the
```

Comment:
- Do two or more boats sound, or have a data structure in a spectrogram that is vastly different from one boat? 
- How often does this happen? (Back of the hand, don't need exact number)
- Would it be a big issue to set this always to 1 in practice? Would that significantly reduce the amount of data collected?

Reply:
No more than three boats are underway for each sample. This maximum is set in the sample-ais.json file, so could be set to one there. With data as of today, this occurs in 32 out of 49 cases, so defaulting to one would result in only 17 exported clips, instead of 49.

Reply:
Thanks for the information Ray